frontend/ui/lib/widgets/trip_view_widget.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:flutter/material.dart';
+import 'package:tripmeout/widgets/place_block_widget.dart';
+
+class TripViewWidget extends StatefulWidget {
+  TripViewWidget({Key key}) : super(key: key);
+  _TripWidgetState createState() => _TripWidgetState();
+}
+
+class _TripWidgetState extends State<TripViewWidget> {
+  @override
+  void initState() {
+    super.initState();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      decoration: BoxDecoration(
+          border: Border.all(
+            color: Colors.black,
+          ),
+          borderRadius: BorderRadius.all(Radius.circular(5))),
+      margin: const EdgeInsets.all(15.0),
+      padding: const EdgeInsets.all(10.0),
+      width: MediaQuery.of(context).size.width * .80,
```

Comment:
I'm not sure what the best practice is, but we might want to consider clipping this at some maximum device-independent width in the future. It's also possible that we'd actually want _more_ width on small screens. (No need to do that here.)

Reply:
Ah yeah, so this is actually why we want to generally stay away from all these fixed size stuff. Most things should naturally layout well using rows and columns with the correct axis alignments, Expanded, Flexible, and Spacer usage. For things that we actually want to shape differently for different screen sizes, we want [LayoutBuilder](https://api.flutter.dev/flutter/widgets/LayoutBuilder-class.html). You can use it for a full page and smaller sub-widgets. But you usually want something that looks like:

```dart
class AdaptiveWidgetX {
  @override
  Widget build(BuildContext context) {
    return LayoutBuilder(
      builder: (context, constraints) {
        if (constraints.maxWidth > someThreshold) {
          return WideWidgetX();
        } else {
          return NarrowWidgetX();
        }
      }
    );
  }
}
```

Reply:
Should I make this change to this PR or wait until the following?